Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingMiniEngineSample/ModelViewer.cpp

```diff
@@ -1096,25 +1097,27 @@ void D3D12RaytracingMiniEngineSample::Update( float deltaT )
       rayTracingMode = RTM_DIFFUSE_WITH_SHADOWRAYS;
     else if(GameInput::IsFirstPressed(GameInput::kKey_7))
       rayTracingMode = RTM_REFLECTIONS;
-	
-	static bool ignoreController = false;
-	
-	if (GameInput::IsFirstPressed(GameInput::kKey_left))
-	{
-		ignoreController = true;
-		m_CameraPosArrayCurrentPosition = (m_CameraPosArrayCurrentPosition + 1) % m_NumCameraPositions;
-	}
-	else if (GameInput::IsFirstPressed(GameInput::kKey_right))
-		ignoreController = false;
-
-	if (ignoreController)
-	{
-		m_CameraController->UpdateToPosition(m_CameraPosArray[m_CameraPosArrayCurrentPosition].position, m_CameraPosArray[m_CameraPosArrayCurrentPosition].heading, m_CameraPosArray[m_CameraPosArrayCurrentPosition].pitch);
-	}
-	else
-	{
-		m_CameraController->Update(deltaT);
-	}
+    
+    static bool ignoreController = false;
+    
+    if (GameInput::IsFirstPressed(GameInput::kKey_left))
```

Comment:
Map this to a designated unused key. For example "c" to switch between frozen camera and not.